sdk/keyvault/keyvault-admin/test/public/utils/authentication.ts

```diff
@@ -42,47 +38,62 @@ export async function authenticate(
       BLOB_STORAGE_URI: "https://uri.blob.core.windows.net/",
       CLIENT_OBJECT_ID: "01ea9a65-813e-4238-8204-bf7328d63fc6",
     },
-    customizationsOnRecordings: [
-      (recording: any): any =>
-        recording.replace(/"access_token":"[^"]*"/g, `"access_token":"access_token"`),
-      (recording: any): any =>
-        suffix === "" ? recording : recording.replace(new RegExp(suffix, "g"), ""),
-      (recording: any): any =>
-        recording.replace(
-          /keyvault_name\.[a-z-]+\.azure[a-z-]*\.net/g,
-          `keyvault_name.managedhsm.azure.net`
-        ),
-      (recording: any): any => {
-        for (const uuid of generatedUUIDs) {
-          recording = recording.replace(
-            new RegExp(uuid, "g"),
-            "b36b00af-89c6-435f-a43d-9a3087015c27"
-          );
-        }
-        return recording;
-      },
-    ],
-    queryParametersToSkip: [],
+    sanitizerOptions: {
+      generalSanitizers: [
+        {
+          target: `"access_token": "[^"]*"`,
```

Comment:
we shouldn't need this?
since the AAD traffic is out

Reply:
yeah oops, I thought it might still be needed for the challenge-based auth, but it doesn't look like there's anything in the recordings to justify it.